commands/addons.js

```diff
@@ -89,7 +89,7 @@ function* addonGetter(api, app) {
 }
 
 function displayAll(addons) {
-    addons = _.sortByAll(addons, 'app.name', 'plan.name', 'addon.name');
+    addons = _.sortBy(addons, 'app.name', 'plan.name', 'addon.name');
```

Comment:
Maybe it accepts both, but the docs imply that `_.sortBy` takes an array of fields as the second argument when sorting by multiple fields, not var args